Source/RapyutaSimulationPlugins/Private/Core/RRBaseActor.cpp

```diff
@@ -58,10 +58,6 @@ void ARRBaseActor::PreInitializeComponents()
     check(GameSingleton);
 }
 
-// (NOTE) This method, if being called, could only go with a RRGameMode-inheriting game mode setup!
-// Currently, ARRROS2GameMode & ARRGameMode are separate ones.
-// & Maps of ARRROS2GameMode do NOT YET have actors invoking this method.
-// It is up to the Child class, [Initialize()] could be run inside [BeginPlay()] or some place else in advance!
```

Comment:
Are these comments obsolete for this function? 
Otherwise they may be useful to avoid future bad usage?

Reply:
It is moved to header file